### ISSUE TITLE
Duplicate columns in varsToKeep throws error

### ISSUE THREAD

**Issue by hongooi73** [RevolutionAnalytics/dplyrXdf#34]
```r
mtx <- rxDataStep(mtcars, "mtcars.xdf", overwrite=TRUE)

rxDataStep(mtx, varsToKeep=c("mpg", "cyl", "cyl"))

#Failed to allocate 1804812534147383619 bytes.
#Caught exception in file: CxDataFile.cpp, line: 3190. ThreadID: 27540 Rethrowing.
#Caught exception in file: CxAnalysis.cpp, line: 6666. ThreadID: 27540 Rethrowing.
#Caught exception in file: CxAnalysis.cpp, line: 5837. ThreadID: 27540 Rethrowing.
#Caught exception in file: CxAnalysis.cpp, line: 5386. ThreadID: 27540 Rethrowing.
#Error in doTryCatch(return(expr), name, parentenv, handler) : 
#  bad allocation
```

**Comment by hongooi73** [comment]
TFS item 78065

**Comment by hongooi73** [comment]
Fixed in 9.2
